yourapp/README.md

```diff
@@ -0,0 +1,18 @@
+### Steps to run this sample:
```

Comment:
What is the general purpose of this sample over the hello world sample? If it's just for docs purpose and not for user sample purpose, I wonder if it should live elsewhere. Or can you adapt the hello world sample?

Also, we name all of our sample folders according to what the sample demonstrates. We should do the same here.